src/go/rpk/pkg/cli/cmd/wasm/template/wasm.go

```diff
@@ -9,26 +9,46 @@
 
 package template
 
-const wasmJs = `const { SimpleTransform, PolicyError } = require("@vectorizedio/wasm-api");
+const wasmJs = `const {
+  SimpleTransform,
+  PolicyError,
+  calculateRecordLength,
+  calculateRecordBatchSize,
+} = require("@vectorizedio/wasm-api");
 const transform = new SimpleTransform();
 /* Topics that fire the transform function */
-transform.subscribe([
-  /*\"input topics\"*/
-]);
+transform.subscribe(["produce"]);
 /* The strategy the transform engine will use when handling errors */
 transform.errorHandler(PolicyError.SkipOnFailure);
+/* Auxiliar transform function for records */
+const uppercase = (record) => {
+  const newRecord = {
+    ...record,
+    value: record.value.map((value) => {
+      if (value > 97 && value < 122) {
```

Comment:
There's `record.value` but you're also naming the func arg `value`. I think that's confusing.
If I understand correctly, `value` is a char. Why not name it `char`?

Reply:
sounds good

Reply:
Done